easybuild/easyconfigs/e/EigenExa/EigenExa-2.11-intel-2020b.eb

```
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'EigenExa'
+version = '2.11'
+
+homepage = 'https://www.r-ccs.riken.jp/labs/lpnctrt/projects/eigenexa/'
+description = """EigenExa, a part of KMATHLIB, is a high performance eigen-solver."""
+
+toolchain = {'name': 'intel', 'version': '2020b'}
+toolchainopts = {'usempi': True, 'openmp': True}
+
+source_urls = ['https://www.r-ccs.riken.jp/labs/lpnctrt/projects/eigenexa/']
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['87dee8ac13f410a007e82df2688fa7f143883229dac729fd20836f4a28fac43d']
+
+preconfigopts = './cleanup && ./bootstrap && '
+
+parallel = 1
+
+sanity_check_paths = {
+    'files': ['lib/libEigenExa.a'],
```

worth adding a check for the shared library as well
```suggestion
    'files': ['lib/libEigenExa.%s' % x for x in ['a', SHLIB_EXT]],
```